snap/snapcraft.yaml

```diff
@@ -178,10 +184,6 @@ parts:
         licenses/LICENSE-crypt-library
       usr/share/doc/mongodb-mongosh/LICENSE-mongosh:
         licenses/LICENSE-mongosh
-  percona-mongodb-exporter:
-    plugin: dump
-    source: "https://github.com/percona/mongodb_exporter/releases/download\
-      /v0.40.0/mongodb_exporter-0.40.0.linux-64-bit.deb"
```

Comment:
since we removed this and did not add an alternate step where the mongodb-exporter package is installed, the exporter package will not be included in the snap

please convert this step to install the `mongodb-exporter` package (see `mongo-deb` step to see how the `percona-server-mongodb` is installed)